openfasoc/common/drc-lvs-check/magic_commands.tcl

```diff
@@ -8,6 +8,13 @@
 #   tech load $::env(TECH)
 # }
 
+# gds flatglob commands removes the drc errors caused by hierarchy that magic can't process.
+# gds faltglob *capacitor_test_nf* is specific to ldo-gen
```

Comment:
@lulu9312 what type of errors are these?

Reply:
@msaligane This errors are caused because hierarchy structure was incompatible with magic.

Reply:
![vref_block_drc_error](https://user-images.githubusercontent.com/61928883/204095480-71299211-320d-451f-bea5-24ce03332e4d.png)
This were significant errors seen in vref_block